.gitignore

```diff
@@ -65,3 +65,5 @@ buck-out/
 
 # Typedoc
 /docs
+
+.env*
```

Comment:
Minor style nitpick: Trailing newlines can be inserted automatically on save with

```json
"files.insertFinalNewline": true
```

in VSCode, they play better with command-line tools like `cat` or `wc`.

Reply:
Inserted trailing newlines in files where they were missing.